test/spawn.jl

```diff
@@ -309,6 +309,7 @@ let out = Pipe(), echo = `$exename --startup-file=no -e 'print(STDOUT, " 1\t", r
     @test isempty(read(out))
     @test eof(out)
     @test desc == "Pipe(open => active, 0 bytes waiting)"
+    wait(t)
```

Comment:
this line actually fixes a bug unrelated to the PR (we need to wait on all `@async` blocks created in the tests to propagate the failure status)

yes, the other line demonstrates the expected behavioral change